pages/linux/rtcwake.md

```diff
@@ -6,15 +6,15 @@
 
 `sudo rtcwake -m show -v`
 
-- Suspend to ram and wakeup after 10 seconds:
+- Suspend to RAM and wakeup after 10 seconds:
 
 `sudo rtcwake -m mem -s {{10}}`
 
 - Suspend to disk (higher power saving) and wakeup 15 minutes later:
 
 `sudo rtcwake -m disk --date +{{15}}min`
 
-- Freeze the system (more efficient than suspend-to-ram but linux > 3.9 required) and wakeup at a given date and time:
+- Freeze the system (more efficient than suspend-to-RAM but Linux > 3.9 required) and wakeup at a given date and time:
```

Comment:
What do you think about this change?
```suggestion
- Freeze the system (more efficient than suspend-to-RAM but Linux >3.9 required) and wakeup at a given date and time:
```

Reply:
Im personally 50/50. The ` > ` fits better mathematically and grammatically, however `>3.9` is overall better for the situation since it highlights it better.

Also what is Linux 3.9... (ik its the kernel). I really dislike that usage, since their are so many distros and to make it seem like a release annoys me

Reply:
Maybe add a reference to Linux being a kernel as well. Maybe Linux kernel ... >

Reply:
It's a good usage actually — because it was the _Linux kernel_ (i.e. https://github.com/torvalds/linux) that added support for that, not any particular distro (or GUI).